internal/lifecycle/readme.go

```diff
@@ -143,15 +143,15 @@ func extractCodeBlocks(scanner *bufio.Scanner) ([]codeBlock, error) {
 		if strings.Contains(line, codeTag) {
 			if s := scanner.Scan(); !s {
 				if err := scanner.Err(); err != nil {
-					return nil, fmt.Errorf("[lifecycle.codeBlocks] bufio.Scanner: %w", err)
+					return nil, fmt.Errorf("bufio.Scanner.Scan: %w", err)
 				}
-				return nil, fmt.Errorf("[lifecycle.codeBlocks]: unexpected EOF; file ended immediately after code tag")
+				return nil, fmt.Errorf("unexpected EOF: file ended immediately after code tag")
 			}
 
 			startCodeBlockLine := scanner.Text()
 			m := mdCodeFenceStartRegexp.MatchString(startCodeBlockLine)
 			if !m {
-				return nil, fmt.Errorf("[lifecycle.codeBlocks]: expecting start of code block immediately after code tag")
+				return nil, fmt.Errorf("expecting start of code block immediately after code tag")
```

Comment:
These errors might benefit from adding which line of parsing they failed on; I think they'd be confusing otherwise. Can be accessed via https://golang.org/pkg/text/scanner/#Scanner.Pos looks like.

Reply:
Resolved offline, thank you. We're printing the line number during `extractCodeBlocks` and dumping the entire code block elsewhere.